Makefile

```diff
@@ -1,12 +1,25 @@
+export CGO_ENABLED ?= 0
+
+VERSION ?= $(shell git describe --tags 2>/dev/null | sed -e 's/^v//g' || echo "dev")
+REVISION ?= $(shell git rev-parse --short HEAD || echo "unknown")
+DATE ?= $(shell date +%Y-%m-%dT%H:%M:%S%:z)
+LDFLAGS = -s -w \
+          -X github.com/gomeeseeks/meeseeks-box/version.Version=$(VERSION) \
+          -X github.com/gomeeseeks/meeseeks-box/version.Commit=$(REVISION) \
+          -X github.com/gomeeseeks/meeseeks-box/version.Date=$(DATE)
```

Comment:
This, and the docker image creation (darwin, amd64-linux and armv6) are handled with goreleaser. I would like to keep it this way. This is why the makefile is so simple.

I would suggest you to try to run `make snapshot` after getting [goreleaser](https://goreleaser.com/) installed and see how we can leverage this tool as the building and releasing one.

One of the core benefits of using goreleaser is that there is an open issue to [make it provide multi arch docker images](https://github.com/goreleaser/goreleaser/issues/530), which will simplify distribution quite a lot.